src/main/java/com/joyent/manta/monitor/Application.java

```diff
@@ -159,8 +161,19 @@ private static void copyContextToException(final Injector injector,
         });
 
         String message = String.format("%s %d. Additional context is as follows:%s",
-                s, configuration.getJettyServerPort(), messageBuffer);
+                s, jettyServerPort, messageBuffer);
 
         LOG.error(message, e);
     }
+
+    private static int validateJettyServerPort(final String jettyServerPortEnvVariable) {
+        if (jettyServerPortEnvVariable == null) {
+            throw new RuntimeException("Manta Monitor requires env variable JETTY_SERVER_PORT");
+        }
+        if (Integer.parseInt(jettyServerPortEnvVariable) <= 0) {
+            throw new RuntimeException("Jetty server port must be greater than 0");
```

Comment:
I would throw an `IllegalArgumentException` and not a `RuntimeException` here. Runtime exceptions are super generic and generally bad form to be used as-is.

Reply:
Got it.